<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ivo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/icons/icons/stupid_logo.png</normaloff>:/icons/icons/stupid_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frameLeft">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
      <widget class="QFrame" name="frameRight">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoad_Model"/>
    <addaction name="actionOpen_obj"/>
    <addaction name="actionLoad_Texture"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Sheets"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionToggle_Lighting"/>
    <addaction name="actionShow_Grid"/>
    <addaction name="actionZoom_fit"/>
    <addaction name="actionZoom_2D"/>
    <addaction name="actionZoom_3D"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>E&amp;dit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionAutoPack"/>
    <addaction name="actionScale"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Model"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionOpen_obj"/>
   <addaction name="actionLoad_Texture"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionModeMove"/>
   <addaction name="actionModeRotate"/>
   <addaction name="actionModeSnap"/>
   <addaction name="actionModeFlaps"/>
   <addaction name="separator"/>
   <addaction name="actionModeAddSheet"/>
   <addaction name="actionModeMoveSheet"/>
   <addaction name="actionModeRemSheet"/>
   <addaction name="separator"/>
   <addaction name="actionExport_Sheets"/>
   <addaction name="actionScale"/>
   <addaction name="actionSettings"/>
   <addaction name="actionPolypaint"/>
  </widget>
  <action name="actionOpen_obj">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Document-new.png</normaloff>:/icons/icons/48px-Document-new.png</iconset>
   </property>
   <property name="text">
    <string>Import &amp;Model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Document-save-as.png</normaloff>:/icons/icons/48px-Document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Document-save.png</normaloff>:/icons/icons/48px-Document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-System-log-out.png</normaloff>:/icons/icons/48px-System-log-out.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionLoad_Texture">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Image-x-generic.png</normaloff>:/icons/icons/48px-Image-x-generic.png</iconset>
   </property>
   <property name="text">
    <string>Mate&amp;rial Manager</string>
   </property>
   <property name="toolTip">
    <string>Browse current model's materials and assign textures</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/Help-browser.png</normaloff>:/icons/icons/Help-browser.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionModeRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-View-refresh.png</normaloff>:/icons/icons/48px-View-refresh.png</iconset>
   </property>
   <property name="text">
    <string>ModeRotate</string>
   </property>
   <property name="toolTip">
    <string>Rotate mode</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionModeSnap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Go-bottom.png</normaloff>:/icons/icons/48px-Go-bottom.png</iconset>
   </property>
   <property name="text">
    <string>ModeSnap</string>
   </property>
   <property name="toolTip">
    <string>JoinBreak edges mode</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionModeMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/Move_mode.png</normaloff>:/icons/icons/Move_mode.png</iconset>
   </property>
   <property name="text">
    <string>ModeMove</string>
   </property>
   <property name="toolTip">
    <string>Move mode</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionModeFlaps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/Flaps_mode.png</normaloff>:/icons/icons/Flaps_mode.png</iconset>
   </property>
   <property name="text">
    <string>ModeFlaps</string>
   </property>
   <property name="toolTip">
    <string>Switch flaps</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionModeAddSheet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/Sheet-Add.png</normaloff>:/icons/icons/Sheet-Add.png</iconset>
   </property>
   <property name="text">
    <string>ModeAddSheet</string>
   </property>
   <property name="toolTip">
    <string>Add sheet of paper</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="actionModeMoveSheet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/Sheet-Move.png</normaloff>:/icons/icons/Sheet-Move.png</iconset>
   </property>
   <property name="text">
    <string>ModeMoveSheet</string>
   </property>
   <property name="toolTip">
    <string>Move sheet of paper</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </action>
  <action name="actionModeRemSheet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/Sheet-Rem.png</normaloff>:/icons/icons/Sheet-Rem.png</iconset>
   </property>
   <property name="text">
    <string>ModeRemSheet</string>
   </property>
   <property name="toolTip">
    <string>Delete sheet of paper</string>
   </property>
   <property name="shortcut">
    <string>7</string>
   </property>
  </action>
  <action name="actionExport_Sheets">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Printer.png</normaloff>:/icons/icons/48px-Printer.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xport Sheets</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Preferences-system.png</normaloff>:/icons/icons/48px-Preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>Sett&amp;ings</string>
   </property>
   <property name="toolTip">
    <string>Edit rendering, paper and export settings</string>
   </property>
  </action>
  <action name="actionZoom_fit">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-View-zoom-fit.png</normaloff>:/icons/icons/48px-View-zoom-fit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom fit</string>
   </property>
  </action>
  <action name="actionZoom_2D">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-View-zoom-fit.png</normaloff>:/icons/icons/48px-View-zoom-fit.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;2D</string>
   </property>
  </action>
  <action name="actionZoom_3D">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-View-zoom-fit.png</normaloff>:/icons/icons/48px-View-zoom-fit.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;3D</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Edit-undo.png</normaloff>:/icons/icons/48px-Edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Edit-redo.png</normaloff>:/icons/icons/48px-Edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionLoad_Model">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Document-open.png</normaloff>:/icons/icons/48px-Document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/Scale.png</normaloff>:/icons/icons/Scale.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Scale</string>
   </property>
  </action>
  <action name="actionAutoPack">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Package-x-generic.svg.png</normaloff>:/icons/icons/48px-Package-x-generic.svg.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Auto Pack</string>
   </property>
   <property name="toolTip">
    <string>Automatically arrange groups in 2D window</string>
   </property>
  </action>
  <action name="actionShow_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Antialias_Icon.svg.png</normaloff>:/icons/icons/48px-Antialias_Icon.svg.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Show Grid</string>
   </property>
   <property name="toolTip">
    <string>Show grid in 3D view window</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionToggle_Lighting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/light_bulb.png</normaloff>:/icons/icons/light_bulb.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Toggle Lighting</string>
   </property>
   <property name="toolTip">
    <string>Toggle Lighting</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionPolypaint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Polypaint</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
